MinimedKit/PumpManager/MinimedPumpManager.swift

```diff
@@ -205,17 +192,36 @@ public class MinimedPumpManager: RileyLinkPumpManager, PumpManager {
     }
     
     // MARK: - PumpManager
+    private enum SuspendTransition {
+        case suspending
+        case resuming
+    }
+
+    // TODO: Accessed and set on different threads
```

Comment:
Any reason to hold up this TODO? (I assume the action item is synchronizing access to `suspendTransition`)

Reply:
This is coming in a separate PR.